src/main/java/uk/gov/hmcts/reform/pip/data/management/service/artefact/ArtefactDeleteService.java

```diff
@@ -176,6 +181,25 @@ public String deleteArtefactByLocation(Integer locationId, String provenanceUser
         }
     }
 
+    public String deleteAllArtefactsWithLocationNamePrefix(String prefix) {
```

Comment:
We also need to delete the blobs from channel management

Reply:
It is now deleting blos :)